### PR TITLE
Add docker steps for eland model upload

### PR DESCRIPTION
**Pull request by benwtrent**

### CODE REVIEW

File: docs/guide/machine-learning.asciidoc
Comment (droberts195) [inline docs/guide/machine-learning.asciidoc:91]:
```suggestion
[discrete]
[[ml-nlp-pytorch-docker]]
==== Import model with Docker
```

Maybe my exact suggestion here is wrong, but it definitely shouldn't use `#` as the heading marker.

File: docs/guide/machine-learning.asciidoc
Comment (droberts195) [inline docs/guide/machine-learning.asciidoc:95]:
```suggestion
If you want to use Eland without installing it, in order to just run the available scripts, can build the Docker
```

File: docs/guide/machine-learning.asciidoc
Comment (droberts195) [inline docs/guide/machine-learning.asciidoc:97]:
```suggestion
If you want to use Eland without installing it, in order to just run the available scripts, you can build the Docker
```

File: docs/guide/machine-learning.asciidoc
Comment (szabosteve) [inline docs/guide/machine-learning.asciidoc:95]:
```suggestion
IMPORTANT: To use the Docker container, you need to clone the eland repository: https://github.com/elastic/eland
```

File: docs/guide/machine-learning.asciidoc
Comment (szabosteve) [inline docs/guide/machine-learning.asciidoc:98]: It might be a bit easier to read this way.
```suggestion
If you want to use Eland without installing it, you can build the Docker container to run the available scripts:
```

File: docs/guide/machine-learning.asciidoc
Comment (szabosteve) [inline docs/guide/machine-learning.asciidoc:110]:
```suggestion
Running installed scripts is also possible without an interactive shell, for example:
```

File: docs/guide/machine-learning.asciidoc
Comment (szabosteve) [inline docs/guide/machine-learning.asciidoc:120]:
```suggestion
Replace the `$ELASTICSEARCH_URL` with the URL for your Elasticsearch cluster. For authentication purposes, include an administrator username and password in the URL in the following format: `https://username:password@host:port`.
```

**Review by droberts195 — APPROVED** [review]
LGTM if you could just apply Istvan's suggestions before merging

File: docs/guide/machine-learning.asciidoc
Comment (benwtrent) [inline docs/guide/machine-learning.asciidoc:98]:
```suggestion
If you want to use Eland without installing it, you can build the Docker container to run the available scripts:
```

**Pull request merged**
